src/shared/components/ui/List/List.js

```diff
@@ -35,14 +36,19 @@ List.displayName = 'List'
 List.propTypes = {
   children: PropTypes.node.isRequired,
   className: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
-  listStyle: PropTypes.string,
-  variant: PropTypes.oneOf(['inline'])
+  listStyle: PropTypes.oneOf([
+    'ordered-check',
+    'ordered-decimal',
+    'ordered-disc',
+    ''
```

Comment:
czemu pusty string?

Reply:
To samo listStyle jest w defaultProps pustym stringiem, a proptype był dowolnym stringiem, co pozwalało przekazać dowolny string niekoniecznie jeden z dostepnych listStyle. Można to zmienić, tylko zrobienie nulla lub undefined też trzeba uwzględnić w PropTypes.oneOf bo tak wywali error :)